variables.tf

```
@@ -67,3 +55,52 @@ variable "config_logs_prefix" {
   default     = "config"
   type        = "string"
 }
+
+# Service Switches
+variable "default_enable" {
+  description = "Create one bucket with all services as bucket keys."
```

The description should convey that this is a default that can be overridden. Something like:

```suggestion
  description = "Whether all services should be enabled by default. Individual services can override this default."
```